### ISSUE TITLE
Connecting to rundeck to fetch jobs .. fails with "closed"

### ISSUE THREAD

**Issue by clemensutschig** [opendevstack/ods-provisioning-app#261]
**Describe the bug**

Whitelabel Error Page
This application has no explicit mapping for /error, so you are seeing this as a fallback.

Thu Oct 31 16:00:16 GMT 2019
There was an unexpected error (type=Internal Server Error, status=500).
closed

**To Reproduce**
provide wrong rundeck authentication means / token

**Expected behavior**
A clear error message coming from the server, containing the cause!

**Affected version (please complete the following information):**
 - OpenDevStack [1.2.0]

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```
  | 2019-10-31 16:00:16 DEBUG o.s.s.w.c.SecurityContextPersistenceFilter - SecurityContextHolder now cleared, as request processing completed
  | 2019-10-31 16:00:16 ERROR o.a.c.c.C.[.[.[.[dispatcherServlet] - Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is java.lang.IllegalStateException: closed] with root cause
  **| java.lang.IllegalStateException: closed**
  | at okio.RealBufferedSource.rangeEquals(RealBufferedSource.java:398)
  | at okio.RealBufferedSource.rangeEquals(RealBufferedSource.java:392)
  | at okhttp3.internal.Util.bomAwareCharset(Util.java:431)
  | at okhttp3.ResponseBody.string(ResponseBody.java:174)
  | at org.opendevstack.provision.util.rest.RestClient.sendPreAuthRequest(RestClient.java:83)
  | at org.opendevstack.provision.util.rest.RestClient.execute(RestClient.java:43)
  | at org.opendevstack.provision.services.RundeckAdapter.getJobsFromServer(RundeckAdapter.java:254)
  | at org.opendevstack.provision.services.RundeckAdapter.initializeJobsForGroup(RundeckAdapter.java:152)
  | at org.opendevstack.provision.services.RundeckAdapter.getQuickstarters(RundeckAdapter.java:116)
  | at org.opendevstack.provision.controller.DefaultController.provisionProject(DefaultController.java:98)
  | at sun.reflect.GeneratedMethodAccessor101.invoke(Unknown Source)
```

**Additional context**

Problem is the double string read ... :(
https://github.com/opendevstack/ods-provisioning-app/blob/master/src/main/java/org/opendevstack/provision/util/rest/RestClient.java#L81-L83

and solution - 
https://teamtreehouse.com/community/fatal-exception-okhttp-dispatcher-2

@stefanlack

**Comment by clemensutschig** [comment]
@stefanlack the real Error behind is a Proxy Error..  super weird.. is okhttp  picking up proxy from env?

**Comment by clemensutschig** [comment]
what's really weird - is that a scale down / up fixed the issue.

**Comment by clemensutschig** [comment]
partially closed by #263
